src/index.js

```diff
@@ -37,15 +43,58 @@ window.addEventListener('load', (event) => {
       </div>
     `;
   });
-
   // Add all the divs to the HTML
-  document.querySelector('#memory-board').innerHTML = html;
+  /** 
+   * Understand the game - done
+   * Flip a card
+   * ---Add clickevent listener on card - done
+   * ---Add the classname turned to the class card in the car - done
+   * Flip only two cards -- done
+   * Compare the two cards -- done
+   * ---If card1 and card2 are not the same flip the cards -- done
+   * ---if Card1 and card2 are the same keep them open -- done
+   *  
+      
+   **/
+
+  document.querySelector("#memory-board").innerHTML = html;
+
+  let cardsFlipped = 0;
+  const cardLimit = 2;
+  let firstFlippedCard;
+  let secondFlippedCard;
 
   // Bind the click event of each element to a function
-  document.querySelectorAll('.card').forEach((card) => {
-    card.addEventListener('click', () => {
-      // TODO: write some code here
-      console.log(`Card clicked: ${card}`);
+  document.querySelectorAll(".card").forEach((card) => {
+    card.addEventListener("click", () => {
+      if (cardsFlipped < cardLimit) {
+        card.classList.add("turned");
+        cardsFlipped++;
+
+        if (cardsFlipped === 1) {
+          firstFlippedCard = card;
+        } else if (cardsFlipped === 2) {
+          secondFlippedCard = card;
+
+          if (
+            firstFlippedCard.dataset.cardName ===
+            secondFlippedCard.dataset.cardName
+          ) {
+            setTimeout(() => {
+              firstFlippedCard.parentNode.removeChild(firstFlippedCard);
+              secondFlippedCard.parentNode.removeChild(secondFlippedCard);
+            }, 1000);
+          } else {
+            setTimeout(() => {
+              firstFlippedCard.classList.remove("turned");
+              secondFlippedCard.classList.remove("turned");
+            }, 1000);
+          }
+          cardsFlipped = 0;
+        }
```

Comment:
You're not making use of the MemoryGame class, which has methods that were created to handle a lot of what you're duplicating in here, and more. That whole score board depends on the MemoryGame class.